src/config/mailConfig.js

```diff
@@ -0,0 +1,39 @@
+import nodemailer from 'nodemailer';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const email = process.env.EMAIL;
+const host = process.env.SMTP_HOST;
+const port = process.env.SMTP_PORT;
+const username = process.env.SMTP_USERNAME;
+const pass = process.env.SMTP_MAIL_PASSWORD;
+
+const transporter = nodemailer.createTransport({
+  host,
+  port,
+  auth: {
+    user: username,
+    pass
+  }
+});
+
+const sendMail = (req, res, body) => {
+  const allRecipientEmail = body.receiver.join(', ');
```

Comment:
De-structure params from body object. 
`const { email } = res.body;`

Reply:
This has been resolved